### DISCUSSION TITLE
Show the full path to the selected span

### DISCUSSION THREAD

**Discussion by gao-artur** [grafana/grafana#69867] [Observability]
**Is your feature request related to a problem? Please describe.**
In large traces, finding the path to the specific span is hard. You need to scroll up to find its parent, then scroll more to find the parent's parent, and so on.

**Describe the solution you'd like**
I'd like to have a full path to the selected span, similar to how it's done in Windows Registry Editor, for example. Ideally, it would be possible to click on any fragment of the path and jump directly to the span representing this fragment. 
![image](https://github.com/grafana/grafana/assets/4518483/90706eaa-c016-45a4-bb3b-5680e814916e)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82643. Feel free to continue the discussion around this there. Thank you!
